### PR TITLE
Make cookies secure

### PR DESCRIPTION
**Pull request by imnutz**

### CODE REVIEW

File: lib/utils/setCookie.js [hunk 1]
```diff
@@ -18,13 +18,17 @@ module.exports = function setCookie (storage, name, value) {
...
+
   var expires = new Date()
   expires.setSeconds(expires.getSeconds() + clone.expires)
 
   // When it's localhost, an IP, or custom domain, set the cookie directly
-  if (is.ip || is.local || is.custom) {
-    clone.domain = is.local ? null : clone.domain
-    cookie.setItem(name, value, expires, clone.path, clone.domain)
+  if (is.local) {
+    cookie.setItem(name, value, expires, clone.path, null)
```
Comment (tung-vu-td) [inline lib/utils/setCookie.js:29]: IMO, for consitency with the later cookie.setItem call:
`cookie.setItem(name, value, expires, clone.path, false, 'None')`

File: lib/utils/setCookie.js [hunk 2]
```diff
@@ -18,13 +18,17 @@ module.exports = function setCookie (storage, name, value) {
...
+  } else if (is.ip || is.custom) {
+    cookie.setItem(name, value, expires, clone.path, clone.domain, true, sameSite)
```
Comment (tung-vu-td) [inline lib/utils/setCookie.js:31]: IMO, for easier read:
`cookie.setItem(name, value, expires, clone.path, true, 'None')`

File: lib/utils/setCookie.js [hunk 3]
```diff
@@ -42,7 +46,7 @@ module.exports = function setCookie (storage, name, value) {
     } else {
       for (; i < ll; i++) {
         clone.domain = domains[i]
-        cookie.setItem(name, value, expires, clone.path, clone.domain)
+        cookie.setItem(name, value, expires, clone.path, clone.domain, true, sameSite)
```
Comment (tung-vu-td) [inline lib/utils/setCookie.js:49]: IMO, for easier read:
`cookie.setItem(name, value, expires, clone.path, true, 'None')`

File: lib/utils/setCookie.js (hunk 1 above)
Comment (imnutz) [reply lib/utils/setCookie.js]: Unfortunately, 'Secure' is always set to `true` when setting sameSite to 'None', for the local we can ignore this. IMHO it is for testing only.

**Pull request merged**